docs/api-auth.md

```diff
@@ -25,30 +27,123 @@ There are three different ways a user is considered an admin:
+
+1. Type `dce auth` in your command terminal. This will open a browser with a log in screen. Enter the username and password for the non-admin user that you created. Reset the password as prompted.
+
+    ![Quick start user login](./img/quickstartuserlogin.png)
+
+1. Upon successfully logging in, you will be redirected to a credentials page containing a temporary authentication code. Click the button to copy the auth code to your clipboard.
+
+    ![Credentials Page](./img/credspage.png)
 
+1. Return to your command terminal and paste the auth code into the prompt.
 
+    ```
+    dce auth
+    ✔ Enter API Token: : █
+    ```
 
-### IAM Policy for DCE API requests
+1. You are now authenticated as a DCE User. Test that you have proper authorization by typing `dce leases list`.
+This will return an empty list indicating that there are currently no leases which you can view.
+If you are not properly authenticated as a user, you will see a permissions error.
 
-The IAM principal used to send requests to the DCE API must have sufficient permissions to execute API requests.
+    ```
+    dce leases list
+    []
+    ```
 
-The Terraform module in the repo provides an IAM policy with appropriate permissions for executing DCE API requests. The policy name and ARN are available as Terraform outputs.
+1. Users are not authorized to list child accounts in the accounts pool. Type `dce accounts list` to verify that you get a permissions error when trying to
+view information you do not have access to.
 
+    ```
+    dce accounts list
+    err:  [GET /accounts][403] getAccountsForbidden
+    ```
+
+1. You will need to be authenticated as an admin before continuing to the next section. Type `dce auth` to log in as a different user. Sign out, then enter the username
+and password for the admin that you created. As before, copy the auth code and paste it in the prompt in your command terminal.
+
+    ![Admin login](./img/quickstartadminlogin.png)
+
+1. Test that you have admin authorization by typing `dce accounts list`. You should see an empty list now instead of a permissions error.
+
+    ```
+    dce accounts list
+    []
+    ```
+
+## Using IAM Credentials
+
+The DCE API accepts authentication via IAM credentials using [SigV4 signed requests](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html). 
+
+At minumum, the IAM credentials used to access DCE must have an attached policy that grants permission to invoke the DCE API Gateway methods, e.g.
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "execute-api:Invoke"
+      ],
+      "Resource": [
+        "${aws_api_gateway_rest_api.gateway_api.execution_arn}/*"
+      ]
+    }
+  ]
+}
+JSON
+}
 ```
-terraform output api_access_policy_name
-terraform output api_access_policy_arn
-```
\ No newline at end of file
+
+This policy is accessible via the api_access_policy_name & api_access_policy_arn terraform outputs.
```

Comment:
Not necessarily explanatory how to access these. Maybe link to /terraform.html#accessing-terraform-outputs